cscs-checks/system/slurm/slurm_check.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+# TODO: sinfo, check if the normal, long, debug, etc... partitions are present?
```

Comment:
I wouldn't look for specific partitions.

Reply:
Done, TODO deleted